SU2_CFD/src/solver_direct_mean_inc.cpp

```diff
@@ -2096,10 +2096,14 @@ void CIncEulerSolver::Source_Residual(CGeometry *geometry, CSolver **solver_cont
     
     for (iPoint = 0; iPoint < nPointDomain; iPoint++) {
       
-      /*--- Load the conservative variables ---*/
+      /*--- Load the primitive variables ---*/
       
-      numerics->SetConservative(node[iPoint]->GetSolution(),
-                                node[iPoint]->GetSolution());
+      numerics->SetPrimitive(node[iPoint]->GetPrimitive(), NULL);
```

Comment:
Why is the second argument here NULL ...

Reply:
As the source term integration uses cellwise (node i) constant data, the data for the 'node j' is not necessary for source terms. For density of 'node j' I now changed it to `0.0` to express clearly that that it is not necessary. To answer the question: I guess there was no bigger thought behind that... just couldn't think of a scalar alternative to the `NULL` pointer at that moment (still can't tbh) or just copied from the` if (axisymmetric)` part below where it is done as well :)

Reply:
You can also pass iPoint in for both arguments if you want something more clear, but yes, for the sources we don’t use jPoint